apps/condo/domains/notification/constants/constants.js

```diff
@@ -475,7 +478,7 @@ const MESSAGE_META = {
             url: { required: true },
         },
     },
-    [NEWS_ITEM_COMMON_MESSAGE_TYPE]: { ...newsItemMessageMeta },
+    [NEWS_ITEM_COMMON_MESSAGE_TYPE]: { ...newsItemMessageMeta, throttlePeriodForUser: 3600 },
```

Comment:
it looks like a part of MESSAGE_DELIVERY_OPTIONS

Reply:
Done